packages/bff/src/server.ts

```diff
@@ -109,7 +108,6 @@ const startServer = async (): Promise<void> => {
     appConfigConnectionString,
   });
   server.register(graphqlApi);
-  server.register(graphqlStream);
```

Comment:
skulle vi kanskje ha gjort dette i to steg? Slik at subscriptions i bff er oppe til vi har gjort byttet og så fjerner vi subscription i bff etterpå?

Reply:
ja i alle miljø ja? Ja da er det vel grett